web/src/main/java/com/futurewei/alcor/web/entity/route/RouterWebRequestJson.java

```diff
@@ -15,18 +15,16 @@
 */
 package com.futurewei.alcor.web.entity.route;
 
-public class RouterState {
-    private String tenantId;
+import lombok.Data;
```

Comment:
@kevin-zhonghao Forgot to mention this to you earlier. Going forward, let us not use lombok.Data, instead, let us write setter and getter explicitly.

Reply:
NP ~ I will remember that and use setter and getter directorly, but in this PR, most of web class used @Data. Should I fix them all?

Reply:
@kevin-zhonghao It is okay. Plan to change it when you get bored :-)